azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -86,6 +87,8 @@
       "azkaban.executorinfo.refresh.maxThreads";
   private static final String AZKABAN_MAX_DISPATCHING_ERRORS_PERMITTED =
       "azkaban.maxDispatchingErrors";
+  private static final String AZKABAN_EVICT_EXECUTIONS =
```

Comment:
Please move user-facing config keys to the Constant class.

Reply:
Ok, will do, sorry.

Reply:
No worry. 

Our strategy is to gradually move the keys to the commonplace as we make changes. Do you mind moving the other keys in this class too? 

Thanks

Reply:
Sure I can move them as well

Reply:
I created a separate PR for this https://github.com/azkaban/azkaban/pull/1660 (with the exception of not adding a new property for eviction)